litmus-portal/frontend/src/views/CreateWorkflow/ChooseWorkflow/index.tsx

```diff
@@ -138,7 +138,7 @@ const ChooseWorkflow = forwardRef((_, ref) => {
           >
             <AccordionSummary>
               <RadioButton value="C" onChange={(e) => handleChange(e)}>
-                {t('createWorkflow.chooseWorkflow.optionC')}
+                {t('createWorkflow.chooseWorkflow.optionC')}{' '}
```

Comment:
is this needed? can &nbsp; be used?

Reply:
Yes, space is not working here.
![Screenshot from 2021-04-14 11-44-37](https://user-images.githubusercontent.com/28081206/114662786-ed3d7000-9d16-11eb-8ea8-66812f1feb5e.png)

Reply:
cc: @Saranya-jena i meant `&nbsp;` since its markdown my earlier comment just had a blank space PTAL

Reply:
Is there any issue with using `{' '}` ? It is just a syntactic element added by the linter to clarify a white space.